README.md

```diff
@@ -25,9 +37,10 @@ AntidoteDB has a simple transactional interface:
 - `commit_transaction/2`
 - `abort_transaction/2`
 
-Here is a detailed interface explanation:
+In the following, you find a short description of the API.
+A more elaborate version with examples can be found in the [AntidoteDB Documentation](https://antidotedb.gitbook.io/documentation/api/protocol-buffer-api).
 
-#### start_transaction(Pid, Timestamp) -> {ok, TxId} | {error, Reason}.
+#### Starting a transaction
```

Comment:
If you remove the signature here, the sentence below starting with "This function" does not fit. Maybe add the signature as a code snippet?

(same below)

Reply:
Done